frontend/trip-planner/src/Components/TripCard.js

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import { Link, useNavigate } from 'react-router-dom';
+import { useState } from "react";
+import TripAddUpdate from './TripAddUpdate';
+import dateFormat, { masks } from "dateformat";
+import {deleteSpecificTrip} from '../Api';
+import styles from './Styles/TripCard.css';
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+export default function TripCard(props) {
+  const [showForm, setShowForm] = useState(false);
+  const [deleteFailed, setDeleteFailed] = useState(false);
+  const navigate = useNavigate();
+
+  function onClickEdit(e) {
+    e.preventDefault();
+    setShowForm(!showForm);
+  };
+
+  async function onClickDelete(event) {
+    event.preventDefault();
+    try {
+      const result = await deleteSpecificTrip(props.trip.id);
+      if (result.status === 401) {
+        navigate('/login');
+      } else {
+        if (result.status === 200) {
+          setDeleteFailed(false);
+          props.getUserTrips();
+        } else {
+          setDeleteFailed(true);
+        }
+      }
+    } catch (e) {
+      navigate('/error');
+    }
+  };
+
+  return (
+    <>
+      <Card sx={{ minWidth: 375 }} className='tripCard'>
```

Comment:
Proposal:
I think it will be really impressive if the background of this card will be a photo from the city.
If you are using the `AutoComplete` in `TripAddUpdate`, then it's really easy to save a photo to the database.
Then I suggest the following:
Present the picture on the `Card` background, on top of it put a black `div` with some transparency and white text.
It will look like Royal-Caribbean cards in their home page: https://www.royalcaribbean.com/
I.e.
```
<Card .... style={{backgroundImage: `url(${props.trip.photo})`}}>
  <div className={'tripCardOverlay'}>
     ...
  </div>
</Card>
```
where the class `tripCardOverlay` will have a `100%` width and height, a black background with some transparency (e.g. `rgba(0, 0, 0, 0.5)`), and white text color.